### ISSUE TITLE
Keeps Crashing

### ISSUE THREAD

**Issue by grubstea** [kitsushadow/forgecraft-issues#2]
I love the idea of primalcore and love playing with it.  I wished to play with your mod cause it looks fascinating but it does not seem to load in so here is my crash report.
[crash-2019-06-24_13.58.42-client.txt](https://github.com/kitsushadow/forgecraft-issues/files/3322049/crash-2019-06-24_13.58.42-client.txt)

**Comment by DbpGaming** [comment]
am having the same crash, the last 3 updates on curseforge all have it

**Comment by DbpGaming** [comment]
fixes it self when adding crafttweaker

**Comment by grubstea** [comment]
DbpGaming,

Thank you for the fix.  That needs added to the dependencies.
________________________________

**Comment by kitsushadow** [comment]
It's not a dependecy. Somehow Craft Tweaker isn't being treated properly. I have looked into this multiple times and don't have a resolution YET. It may require some drastic measures but I haven't had the time to try those.

**Comment by jena-th** [comment]
Yup, had this one too, but as I stated in https://github.com/kitsushadow/forgecraft-issues/issues/8 adding CraftTweaker seems to be a sufficient enough workaround for now to have ForgeCraft working ASAP in the meantime that Kitsune have time to fix the code as it should be done (I guess).
